### PR TITLE
Update SumTotal.md

### PR DESCRIPTION
**Pull request by Gareth-Rodrigues**
updating SumTotal .md file with image for visual instruction of Sumtotal Connector from within PowerBI Desktop
added note for mention of scripts popup warning during OAuth login

### CODE REVIEW

File: powerquery-docs/Connectors/SumTotal.md [hunk 1]
```diff
@@ -1,70 +1,65 @@
...
 ## Summary
 
 | Item | Description |
 | ---- | ----------- |
-| Release State | Beta |
-| Products | Power BI (Datasets) |
-| Authentication types | OAuth 2.0 |
-| Function Reference Documentation | - |
+| Release State | General Availability |
+| Products | Power BI (Datasets)<br/>Power BI (Dataflows)<br/>Dynamics 365 Customer Insights |
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/SumTotal.md:12]: This is not accurate - the connector is and only will be able in Power BI (Datasets) for the foreseeable future.

File: powerquery-docs/Connectors/SumTotal.md [hunk 2]
```diff
@@ -1,70 +1,65 @@
...
+| Authentication types | SumTotal credentials via OAuth2.0 |
+| Function Reference Documentation | Web.Conents <br> Table.ToNavigationTable <br>Table.FromRecords <br> Json.Document|
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/SumTotal.md:14]: If this is changed, this should include links to the M function reference for each of these functions

File: powerquery-docs/Connectors/SumTotal.md [hunk 3]
```diff
@@ -1,70 +1,65 @@
...
 ## Capabilities supported
 
-* Import
+* Query Multiple OData endpoints
+* Create rich and visual reporting via query data
+* Advanced
+   * Optionally filter records by RowVersionId parameter to get incremental data
 
-## Finding your SumTotal hosted URL
+## Finding your Sumtotal Hosted URL
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/SumTotal.md:28]: nit/style - should remain "hosted"

**Comment by Gareth-Rodrigues** [comment]
Hello Bob,

 

Sure thing I will take a look and amend where necessary.

Thanks

 

Kind regards 

Gareth Rodrigues | Software Engineer

COMPRSA | Cloud-based Business Solutions

 ***@***.***> ***@***.***|  <http://www.comprsa.com/> comprsa.com

RSA: +2741 368 2299

USA: +1407 815 2747

Skype: ***@***.***

**Comment by Gareth-Rodrigues** [comment]
Hi @bezhan-msft thanks for the heads up that this was still pending, i finally got around to updating the .md as per your feedback provided, do let me know if you see any issues with the updated changes - https://github.com/MicrosoftDocs/powerquery-docs/pull/161/files

Thanks again
Gareth

**Comment by bezhan-msft** [comment]
@DougKlopfenstein can you please take a look?

File: powerquery-docs/Connectors/SumTotal.md [hunk 4]
```diff
@@ -1,70 +1,65 @@
...
 
 | Item | Description |
 | ---- | ----------- |
-| Release State | Beta |
+| Release State | General Availability |
 | Products | Power BI (Datasets) |
-| Authentication types | OAuth 2.0 |
-| Function Reference Documentation | - |
+| Authentication types | SumTotal credentials via OAuth2.0 |
+| Function Reference Documentation | [Web.Contents](#https://docs.microsoft.com/en-us/powerquery-m/web-contents) <br> [Table.ToNavigationTable](https://docs.microsoft.com/en-us/power-query/helperfunctions#tabletonavigationtable) <br>[Table.FromRecords](https://docs.microsoft.com/en-us/powerquery-m/table-fromrecords) <br> [Json.Document](https://docs.microsoft.com/en-us/powerquery-m/json-document)|
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/SumTotal.md:14]: We don't need this here as there are no new functions being created other than the main connector function.

File: powerquery-docs/Connectors/SumTotal.md [hunk 5]
```diff
@@ -1,70 +1,65 @@
...
+* Query Multiple OData endpoints
+* Create rich and visual reporting via query data
+* Advanced
+   * Optionally filter records by RowVersionId parameter to get incremental data
 
-## Finding your SumTotal hosted URL
+## Finding your Sumtotal hosted URL
 
-Copy the SumTotal hosted root URL in full. This root URL is the unique URL specific to your instance. The URL will be in the format of **https://\<*yourdomain*>.sumtotal.host/**. Make sure not to copy the rest of the URL. Keep this URL somewhere handy so you can use it later.
+Copy the SumTotal hosted root URL in full. This root URL is the unique URL specific to customer. The URL will be in the format of **https://\<*yourcustomerdomain*>.sumtotal.host/**. Make sure not to copy the rest of the URL. Keep this URL somewhere handy so you can use it later.
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/SumTotal.md:30]: to the customer

File: powerquery-docs/Connectors/SumTotal.md [hunk 6]
```diff
@@ -1,70 +1,65 @@
...
 
 To connect to SumTotal from Power BI Desktop:
 
-1. In the **Get Data** experience, select **SumTotal** in the **Other** category, and then select **Connect**.
+1. Select **Get data** from the **Home** tab.
+
+2. In the **Get Data** dialog box, select **Other > SumTotal**, and then select **Connect**.
 
-2. Enter the server URL address of the data you want to load.
+![alt text](https://marketplace.sumtotalsystems.com/Content/Product/38/media/b642fae5-f7f9-49b3-8e16-25c3cb3373ec.png)
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/SumTotal.md:43]: We cannot reference external images from our docs.

**Review by bezhan-msft — CHANGES_REQUESTED** [review]
Left a few comments - Doug, our docs writer, might leave more.

**Comment by Gareth-Rodrigues** [comment]
thank you @bezhan-msft for the reviews, i will take a look and update accordingly

**Comment by DougKlopfenstein** [comment]
Closed because changes were made in private pull request that's documented above.

**Pull request closed without merging**